src/components/selection_input/selection_input.ts

```diff
@@ -143,7 +168,8 @@ export class SelectionInput extends Component<Props, SpreadsheetChildEnv> {
 
   checkChange() {
     const value = this.env.model.getters.getSelectionInputValue(this.id);
-    if (this.previousRanges.join() !== value.join()) {
+    const valid = !this.isInvalid && this.ranges.every((range) => range.isValidRange);
```

Comment:
why isn't `this.ranges.every((range) => range.isValidRange)` part of `isInvalid` then ?

Reply:
For what I understand, isInvalid is coming from the parent (so it should be related to the parent) and is optionnal, so it can be totally not related to the fact a range is a valid range